.gitignore

```diff
@@ -57,7 +57,6 @@ profiles.yml
 crash.log
 *.tfstate
 *.tfstate.backup
-*.lock.hcl
```

Comment:
terraform recommends committing this to git https://developer.hashicorp.com/terraform/language/files/dependency-lock#lock-file-location

> You should include this file in your version control repository so that you can discuss potential changes to your external dependencies via code review